### PR TITLE
changed: manually merge pr's to work around slow updating merge refs

### PR DESCRIPTION
**Pull request by akva2**
This is manually merging the pr branch into master instead of relying on the github refs to do so.

I don't particularly like it, it will be less robust but ...

### CODE REVIEW

**Comment by atgeirr** [comment]
I have to admit I do not understand the tradeoffs at anything but a superficial level. My understanding (which could be wrong...): slow merge ref updates (I do not really know what that is) have caused trouble with triggering Jenkins, and this change moves the makes the whole process more likely to respond quickly, but also makes it less robust somehow. If this reduces the probability and severity of Jenkins snags, or makes the snags more obvious so that a developer can work around them easily by retriggering, I am all for it.

File: jenkins/build-opm-module.sh [hunk 1]
```diff
@@ -130,8 +130,18 @@ function clone_module {
...
+  if [ "$2" == "master" ]
+  then
+    git fetch --depth 1 origin $2:branch_to_build
+    git checkout branch_to_build
+  else
+    git fetch --depth 1 origin master
+    git fetch --depth 1 origin $2:branch_to_build
+    git checkout master
+    git merge branch_to_build
+    test $? -eq 0 || exit 1
```
Comment (joakim-hove) [inline jenkins/build-opm-module.sh]: Is there a reason not to just use:
```sh
#/bin/bash
set -e
```
to capture *all* errors?

**Comment by joakim-hove** [comment]
Apart from the `set - e` suggestion I think this should be merged.

**Comment by joakim-hove** [comment]
Apart from the `set - e` suggestion I think this should be merged.

File: jenkins/build-opm-module.sh (hunk 1 above)
Comment (akva2) [reply jenkins/build-opm-module.sh]: not really in this case. but i like to have control over my exit codes in general..

**Comment by akva2** [comment]
rebased.

**Pull request merged**